README.md

```diff
@@ -1,2 +1,61 @@
-# terraform-aws-eks-auth
-Terraform module for generating AWS EKS configuration
+# Terrafrom module that generates the EKS kubeconfig
+
+Terraform module for generating AWS EKS configuration file and access token
+
+This module re-uses the kubeconfig template and plan snippets from the official EKS module `terraform-aws-modules/eks/aws`
+
+## Usage
+
+```hcl
+data "aws_eks_cluster" "this" {
+  name = "my_eks_cluster"
+}
+
+data "aws_iam_role" "kubernetes_auth_role" {
+  name = "eks_auth_role"
+}
+
+module "eks_auth" {
+  source = "sailthru/terraform-eks-auth/aws"
+  aws_profile = "default"
```

Comment:
should this to be replace by the specific aws account profile?

edit -- i see it is an arg and this is the default value from the input below